.github/workflows/sycl_gen_test_matrix.yml

```diff
@@ -27,13 +27,13 @@ on:
     outputs:
       lts_matrix:
         description: "Generated Matrix"
-        value: ${{ jobs.resolve_matrix.outputs.lts_matrix }}
+        value: ${{ jobs.testing_matrix.outputs.lts_matrix }}
       lts_aws_matrix:
         description: "Generated Matrix AWS subset"
-        value: ${{ jobs.resolve_matrix.outputs.lts_aws_matrix }}
+        value: ${{ jobs.testing_matrix.outputs.lts_aws_matrix }}
 jobs:
-  resolve_matrix:
-    name: Resolve Test Matrix
+  testing_matrix:
+    name: Generate Testing Matrix
```

Comment:
Why do you prefer `testing matrix` to `test matrix`?

Reply:
No specific preference... synonyms to me...
I guess I used "test" where shorter description is preferable and "testing" where longer is fine